shared/settings/nav/index.desktop.js

```diff
@@ -56,6 +56,7 @@ const variantBannerStyle = {
 const itemStyle = {
   ...globalStyles.flexBoxRow,
   height: 40,
+  paddingRight: globalMargins.small,
```

Comment:
I can make the other one a paddingLeft, but a marginRight would give spacing to the text and would move the badge.